### PR TITLE
Adapt rollup in gulpfile and fix failing build process by rollup

### PR DESCRIPTION
**Pull request by magicmatatjahu**
**Description**

Changes proposed in this pull request:
- Add new task in gulpfile for build by rollup appropriate packages
- Refactor some code in gulpfile
- Add missing babel-plugins (bug with failing build process of packages which use rollup)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -60,8 +61,11 @@
   },
   "devDependencies": {
     "@babel/core": "^7.5.5",
+    "@babel/plugin-proposal-async-generator-functions": "^7.2.0",
     "@babel/plugin-proposal-class-properties": "^7.5.5",
+    "@babel/plugin-proposal-object-rest-spread": "^7.5.5",
     "@babel/plugin-transform-runtime": "^7.5.5",
+    "@babel/preset-env": "^7.5.5",
```
Comment (aerfio) [inline package.json]: Sorry about that 😝

**Comment by magicmatatjahu** [comment]
@aerfio I thought about parallel, but we have a packages that must compile and copy to root `node_modules` before another packages, but I will think how do it.

File: packages/odata-render-engine/yarn.lock [hunk 2]
```diff
@@ -2,5287 +2,3 @@
 # yarn lockfile v1
 
 
-"@babel/code-frame@7.0.0-beta.53":
```
Comment (aerfio) [inline packages/odata-render-engine/yarn.lock]: This file should be deleted?

Comment (magicmatatjahu) [reply packages/odata-render-engine/yarn.lock]: I don't know. We don't have this plugin in package.json, so probably when I added new plugins and presets then yarn automatically removed unnecessary packages.

Comment (magicmatatjahu) [reply packages/odata-render-engine/yarn.lock]: Yes, I removed all dependencies. This package has only peerDependencies, so lock will be always empty :P

**Pull request merged**